From dd201ffbb1de4f371f2c9296454e09d67a1b785e Mon Sep 17 00:00:00 2001
From: Codrin Ciubotariu <codrin.ciubotariu@microchip.com>
Date: Fri, 18 Mar 2022 11:26:09 +0200
Subject: [PATCH 032/471] ASoC: atmel: mchp-pdmc: print the correct property
 name

The correct property is 'microchip,mic-pos', not 'mchp,mic-pos', so
replace all occurences of 'mchp,mic-pos' with 'microchip,mic-pos'.
Fix a multi-line comment format while we are at it.

Fixes: 50291652af52 ("ASoC: atmel: mchp-pdmc: add PDMC driver")
Signed-off-by: Codrin Ciubotariu <codrin.ciubotariu@microchip.com>
Link: https://lore.kernel.org/r/20220318092609.130901-1-codrin.ciubotariu@microchip.com
Signed-off-by: Mark Brown <broonie@kernel.org>
---
 sound/soc/atmel/mchp-pdmc.c | 11 ++++++-----
 1 file changed, 6 insertions(+), 5 deletions(-)

diff --git a/sound/soc/atmel/mchp-pdmc.c b/sound/soc/atmel/mchp-pdmc.c
index e0aec5fe8a26..1a7802fbf23c 100644
--- a/sound/soc/atmel/mchp-pdmc.c
+++ b/sound/soc/atmel/mchp-pdmc.c
@@ -879,13 +879,13 @@ static int mchp_pdmc_dt_init(struct mchp_pdmc *dd)
 
 	dd->mic_no = of_property_count_u32_elems(np, "microchip,mic-pos");
 	if (dd->mic_no < 0) {
-		dev_err(dd->dev, "failed to get mchp,mic-pos: %d",
+		dev_err(dd->dev, "failed to get microchip,mic-pos: %d",
 			dd->mic_no);
 		return dd->mic_no;
 	}
 	if (!dd->mic_no || dd->mic_no % 2 ||
 	    dd->mic_no / 2 > MCHP_PDMC_MAX_CHANNELS) {
-		dev_err(dd->dev, "invalid array length for mchp,mic-pos: %d",
+		dev_err(dd->dev, "invalid array length for microchip,mic-pos: %d",
 			dd->mic_no);
 		return -EINVAL;
 	}
@@ -894,9 +894,10 @@ static int mchp_pdmc_dt_init(struct mchp_pdmc *dd)
 
 	dev_info(dd->dev, "%d PDM microphones declared\n", dd->mic_no);
 
-	/* by default, we consider the order of microphones in mchp,mic-pos to
-	 * be the same with the channel mapping; 1st microphone channel 0, 2nd
-	 * microphone channel 1, etc.
+	/*
+	 * by default, we consider the order of microphones in
+	 * microchip,mic-pos to be the same with the channel mapping;
+	 * 1st microphone channel 0, 2nd microphone channel 1, etc.
 	 */
 	for (i = 0; i < dd->mic_no; i++) {
 		int ds;
-- 
2.34.1

